backend/src/routes/comments.js

```diff
@@ -19,4 +19,17 @@ module.exports = async function (fastify) {
       reply.send({ comments });
     },
   });
+
+  fastify.route({
+    method: 'POST',
+    path: '/posts/:postId/comments/:parentId',
+    schema: schema.addComment,
```

Comment:
maybe better to use the method name in schema definition? (postComment, but not addComment)